packages/php-db-import-export/src/Backend/Exasol/ExasolException.php

```diff
@@ -24,7 +24,7 @@ public static function covertException(DBALAlias $e): Throwable
             // strip query from message, there are things like SAS tokens and internal table names
             preg_match('/^(An exception occurred.*?)(Following error.*)=?.\(Session.*/ms', $e->getMessage(), $matches);
 
-            return FileNotFoundException::createFromDbalException($e, $matches[2]);
+            return FileNotFoundException::createFromDbalException($e, $matches[2] ?? '');
```

Comment:
to je asi zvysenim phpka a citlivosti stanu

Reply:
jj on rozumí teď regexom :)